docs/custom-layers.md

```diff
@@ -101,16 +95,11 @@ built-in picking while adding layer specific selection information, override
 `getPickInfo`.
 
 Technical Note: The default picking implementation uses picking colors.
-The layer is rendered
-with a special uniform set to 1, which causes it to render into a
-off-screen framebuffer using `pickingColor` or `instancePickingColor`
-attributes instead of using
-its normal color calculation.
-
-
-See `Layer.encodePickingColor
-
+The layer is rendered with a special uniform set to 1, which causes it to
+render into a off-screen framebuffer using `pickingColor` or
+`instancePickingColor` attributes instead of using its normal color calculation.
 
+See [`Layer.encodePickingColor`](/docs/custom-layers.md#layerencodepickingcolorindex--number).
```

Comment:
I fell like it's better so someone browsing the docs on Github may be able to click be directed to the corresponding method. If the heading changes and we forgot to change this, the link won't work, but there won't be any critical error. It will be transformed to a simple text on the website anyway, since we're not in html5 mode.

But I agree it may be additional work, so it's up to you if you want to remove it.